app-web/src/pages/index.js

```diff
@@ -407,6 +413,7 @@ export const Index = ({
         resultCount={totalSearchResults}
       />
       <Main>
+        {windowHasQuery && siphonResources}
```

Comment:
if search without query, the location.search will be ?q='' which will lead a bug. (cuz windowHasQuery is True)